src/components/slack/attachment/attachment.component.ts

```diff
@@ -11,6 +11,7 @@ import { SettingService } from '../../../services/setting.service';
 export class SlackAttachmentComponent {
     @Input() attachment: Attachment;
     @Input() parser: SlackParser;
```

Comment:
Oops, I didn't check inside the `.html` as removing the `parse` method didn't break `yarn build` (maybe it doesn't check inside htmls).